app/controllers/directory_objects_controller.rb

```diff
@@ -70,11 +70,12 @@ def placard
       name       = person.first + ' ' + person.last
       email = person.email
       department = person.department.title
-      title      = nil
+      title      = person.person_title
```

Comment:
Is 'person_title' a function or the name of a relative? If so, what's wrong with just 'title'?